tests/unit/modules/network/ios/test_ios_acls.py

```diff
@@ -737,6 +769,65 @@ def test_ios_acls_replaced_idempotent(self):
         result = self.execute_module(changed=False)
         self.assertEqual(sorted(result["commands"]), [])
 
+    def test_ios_acls_replaced_changetype(self):
+        self.execute_show_command.return_value = dedent(
+            """\
+            ip access-list extended 110
+                10 permit tcp 198.51.100.0 0.0.0.255 any eq 22 log (tag = testLog)
+                20 deny icmp 192.0.2.0 0.0.0.255 192.0.3.0 0.0.0.255 echo dscp ef ttl eq 10
+                30 deny icmp object-group test_network_og any dscp ef ttl eq 10
+            ip access-list standard test_acl
+                remark remark check 1
+                remark some random remark 2
+            """,
+        )
+        self.execute_show_command_name.return_value = dedent(
+            """\
+            Standard IP access list test_acl
+            """,
+        )
+        set_module_args(
+            dict(
+                config=[
+                    dict(
+                        afi="ipv4",
+                        acls=[
+                            dict(
+                                name="110",
+                                acl_type="standard",
+                                aces=[
+                                    dict(
+                                        grant="deny",
+                                        source=dict(
+                                            address="198.51.100.0",
+                                            wildcard_bits="0.0.0.255",
+                                        ),
+                                    ),
+                                ],
+                            ),
+                            dict(
+                                name="test_acl",
+                                acl_type="standard",
+                                aces=[dict(remarks=["Another remark here"])],
+                            ),
+                        ],
+                    ),
+                ],
+                state="replaced",
+            ),
+        )
+        result = self.execute_module(changed=True)
```

Comment:
Can you please add an integration test for this exact usecase?

Reply:
Done